### PR TITLE
Use Operation ID for tracking ethconnect requests

### PR DESCRIPTION
**Pull request by awrichar**
The Transaction ID is longer-lived, while Operation ID is more appropriate for
a single request (since a transaction may contain multiple operations).

Signed-off-by: Andrew Richardson <andrew.richardson@kaleido.io>

### CODE REVIEW

**Comment by awrichar** [comment]
E2E test failure is fixed by https://github.com/hyperledger/firefly/pull/204 (will update after that is merged)

**Comment by awrichar** [comment]
Updated flow of the operations and transactions involved in pool creation.
PXL_20210924_205830070

File: internal/blockchain/ethereum/ethereum_test.go
Comment (peterbroadhurst) [inline internal/blockchain/ethereum/ethereum_test.go:50]: we've got a safe equals built into UUID you can use (by having a pointer receiver, it works with `nil` pointers on either end - a nice feature of Go)
https://github.com/hyperledger/firefly/blob/25c1039c46820888c2f43eb0bce07feaa4c27fea/pkg/fftypes/uuid.go#L83-L92

**Review by peterbroadhurst — APPROVED** [review]
👍  - thanks for all the re-drawings of the above diagram needed to bring us some order here. 
Think this work and thinking will make a big difference as we go forwards.

Approving with one note for consideration, and also the fact you'll need a merge.

File: internal/blockchain/ethereum/ethereum_test.go
Comment (awrichar) [reply internal/blockchain/ethereum/ethereum_test.go]: I copied this helper from some other tests - seems like it can be removed everywhere.

**Pull request merged**
